README.md

```diff
@@ -101,6 +107,43 @@ TODO: Reading a dataset from pytorch
 - Partitioning for multi-GPU training
 - Local caching
 
+### Reading a dataset from pytorch
+As illustrated in examples/mnist/main.py, reading a petastorm dataset from pytorch can be done via the adapter class `petastorm.pytorch.DataLoader`,
+which allows custom pytorch collating function and transforms to be supplied.
+
+Be sure you have torch and torchvision installed:
+
+```bash
+pip install torchvision
+```
+
+The minimalist example below assumes the definition of a `Net` class and `train` and `test` functions, shown in the mnist example:
+
+```python
+import torch
+from petastorm.pytorch import DataLoader
+
+torch.manual_seed(1)
+device = torch.device('cpu')
+model = Net().to(device)
+optimizer = torch.optim.SGD(model.parameters(), lr=0,01, momentum=0.5)
```

Comment:
is `lr=0,01` a typo?

Reply:
Yeah, good eye! I just posted a fix. Thanks.